scripts/stampy-signed-upload.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const exec = require('child_process').exec;
+
+fs.readdirSync('.')
+  .filter((f) => f.startsWith('sf-v') && f.endsWith('.exe'))
+  .map((f) => ({
+    filename: f,
+    cli: f.split('-')[0],
+    // do it from the end to account for the possibility of the version having a hyphen in it like 2.2.2-beta.0
+    sha: f.split('-').at(-2),
+  }))
+  .map((f) => ({ ...f, shaIndex: f.filename.split('-').indexOf(f.sha) }))
+  // version is the part between the cli and the sha
+  .map((f) => ({ ...f, version: f.filename.split('-').slice(1, f.shaIndex).join('-').replace('v', '') }))
```

Comment:
Why not use regex to grab the version? I think this is difficult to parse

Reply:
I tried, but failed.  This is actually way better looking that where that was headed.